internal/service/rds/cluster_data_source_test.go

```diff
@@ -43,6 +43,40 @@ func TestAccRDSClusterDataSource_basic(t *testing.T) {
 	})
 }
 
+func TestAccRDSClusterDataSource_manage_password(t *testing.T) {
```

Comment:
We should add another AccTest that updates an existing Cluster from self managed master password to RDS managed to ensure a smooth upgrade path.